build/commands/lib/util.js

```diff
@@ -576,7 +576,7 @@ const util = {
       ninjaOpts.push('-j', config.gomaJValue)
     }
 
-    util.run('ninja', ninjaOpts, options)
+    util.run('autoninja', ninjaOpts, options)
```

Comment:
I'm not sure this is what we want, particularly for CI
https://source.chromium.org/chromium/chromium/tools/depot_tools/+/main:autoninja.py;l=165 I think may be a value too large for our goma cluster size
not sure if we want https://source.chromium.org/chromium/chromium/tools/depot_tools/+/main:autoninja.py;l=150 on CI
also what is reclient/reproxy?

Reply:
also the `-j` value from autoninja won't even be used here because we already specify it

Reply:
RE client/proxy will replace the Goma client/server so it's a 2 in 1 - the backend stays the same

Reply:
but then shouldn't this go away? https://github.com/brave/brave-core/pull/10036/files#diff-6149b8e49614376fc44e8cd42d0ce8661e754d14d3bb4df131c3de178e852b0dR575

Reply:
@bridiver the `-j` change has nothing to do with CI or goma - as you've already mentioned, we explicitly specify `-j` for goma, so it won't be affected.

I guess we don't specify `NINJA_BUILD_IN_BACKGROUND` anywhere, so `nice` will not affect CI as well (but can be useful for goma users like me)

This actually improves dev builds. Basically, the story behind the change was that we've noticed our current build script significantly underutilises all 64 cores of @atuchin-m brand new machine, so this simple patch fixed the problem. Personally, I was pretty sure that simple `ninja` properly takes good `-j` but turned out it is not true.

Reply:
right, I forget that I set `-j` for local builds because the default is way too high. So I guess my only concern then is whether we want to `nice` linux builds in CI or should we only use this for dev builds? Do we know what, if any impact it has on CI build times?

Reply:
Wait, underutilizes cores? I have found that reducing the default `-j` value improves build speed and prevents my machine from becoming unusable

Reply:
I'll test a local build using this branch on my mac pro

Reply:
I guess I'm fine approving this and hopefully it does improve build speeds for everyone

Reply:
oh wait, what about the goma startup thing vs reproxy? Don't we need to pick one or the other?

Reply:
would autoninja start reproxy or? we're not using that yet as there's no macOS client

Reply:
I think I mixed things up the first time I looked at it, it will only start reproxy if `use_rbe` is in the args.gn